### PR TITLE
refactor: select dropdown for version selector

### PR DESCRIPTION
**Pull request by aniketkatkar97**
Created a new component to use for the version selector instead of the default select 

https://user-images.githubusercontent.com/51777795/223132050-6417e002-23b3-492a-a272-161bd40779f0.mov

### CODE REVIEW

File: pages/api/getVersionsList.ts [hunk 1]
```diff
@@ -6,7 +6,12 @@ import { ARTICLES_DIRECTORY } from "../../constants/common.constants";
 export default function handler(req, res) {
   try {
     const versionsArray = fs.readdirSync(ARTICLES_DIRECTORY);
-    res.status(200).json(versionsArray);
+    const versionOptionsObj = versionsArray.map((version) => ({
+      label: version,
+      value: version,
+    }));
```
Comment (chirag-madlani) [inline pages/api/getVersionsList.ts:12]: Pull it outside of component defination

Comment (aniketkatkar97) [reply pages/api/getVersionsList.ts:12]: @chirag-madlani The `getVersionsList.ts` is actually an API. So this logic is inside an API handler function.

**Comment by chirag-madlani** [comment]
@aniketkatkar97 Dropdown options are dynamic or manual

**Comment by aniketkatkar97** [comment]
@chirag-madlani, Yes, they are dynamic. The options will populate depending on the content folder.

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**